client/src/js/views/LoginTemplate.html

```diff
@@ -3,8 +3,38 @@
 	<div class="columns">
 	    <div class="header__links">
 		<button class="header__button header__button--last button button--link" id="auth2-signin">ISL Login</button>
-		<a class="header__link" href="#">Fb</a>
-		<a class="header__link" href="#">Tw</a>
+		<a class="header__link" href="#">
+            <svg class="icon--facebook button button--link" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
There's Facebook and Twitter SVG's here and separate files in `/src/img/`, do we need both?